### ISSUE TITLE
caravana-unet downsample issue

### ISSUE THREAD

**Issue by jaideep11061982** [fastai/fastai#1014]
**Describe the bug**

In the file caravana-unet 
 up_p = self.tr_conv(up_p)
        x_p = self.x_conv(x_p)
        cat_p = torch.cat([up_p,x_p], dim=1)
        return self.bn(F.relu(cat_p))
In the original Unet paper convulation is done on the concatenated input of Upsample and convolved feature of block in down path  but above seem to be different than unet...
Can you please explain the logic
**To Reproduce**

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by sgugger** [comment]
I think this is explained in the corresponding course video. If you have any other questions after watching it, you should ask them on the [forum](https://forums.fast.ai/)
